src/utils/constants.ts

```diff
@@ -305,3 +305,7 @@ export const CONTEXT = {
   view: "view",
   edit: "edit"
 };
+export const CONCEPT_GENERAL: string[] = [
+  "Include Retired Concepts ",
+  "Include Added Concepts"
```

Comment:
I would simplify this to just "Include Retired Concepts". I think we're handling Added Concepts in a different way and, in any case, it's a different feature.